lib/backend/k8s/resources/globalnetworkpolicies_test.go

```diff
@@ -116,19 +116,19 @@ var _ = Describe("Global Network Policies conversion methods", func() {
 		BeforeEach(func() {
 			kvp1.Value.(*apiv2.GlobalNetworkPolicy).Spec.DoNotTrack = false
 			kvp1.Value.(*apiv2.GlobalNetworkPolicy).Spec.PreDNAT = true
+			kvp1.Value.(*apiv2.GlobalNetworkPolicy).Spec.ApplyOnForward = true
 			res1.Spec.DoNotTrack = false
 			res1.Spec.PreDNAT = true
-			//TODO: Add back
-			//res1.Spec.ApplyOnForward = true
+			res1.Spec.ApplyOnForward = true
 		})
 
 		AfterEach(func() {
 			kvp1.Value.(*apiv2.GlobalNetworkPolicy).Spec.DoNotTrack = true
 			kvp1.Value.(*apiv2.GlobalNetworkPolicy).Spec.PreDNAT = false
+			kvp1.Value.(*apiv2.GlobalNetworkPolicy).Spec.ApplyOnForward = true
```

Comment:
Not sure I grok what this is doing but should this reverse the change made in `BeforeEach`?

Reply:
ApplyOnForward is always True if either doNotTrack or PreDNAT is true.